packages/integration-sdk/CHANGELOG.md

```diff
@@ -9,6 +9,19 @@ and this project adheres to
 
 ## Unreleased
 
+### Added
+
+- `ExecutionHistory.current: Execution` to provide information about the current
+  execution.
+
+### Changed
+
+- `ExecutionHistory` properties have been renamed as part of adding
+  `current: Execution`, to avoid duplication in the naming: `lastExecution` ->
+  `previous`, `lastSuccessfulExecution` -> `lastSuccessful`.
+- BREAKING: `executeIntegrationInstance` now requires an `ExecutionHistory`
```

Comment:
NOTE: This also breaks `executeWithContext`, which is used by the J1J1 integration. More of an internal detail. People probably aren't using it externally, but it is a public function.

Reply:
Okay, will add note and make this a major version release.